### ISSUE TITLE
Custom logging callback

### ISSUE THREAD

**Issue by roguh** [qntln/aiodebug#3]
I'd like to use this module for JSON logging when an async task blocks the async loop for too long. It'd be nice to have an API that allows users to call a custom callback when a task takes too long to run. For example:

```
aiodebug.log_slow_callbacks.enable(
  0.05,
  callback=lambda duration:
    json_logger.warning("Task blocked async loop for too long", extra={"duration": duration})
)
```

**Comment by vitawasalreadytaken** [comment]
Hi, this is a good idea 👍 

I'll implement this some time this week.

**Comment by roguh** [comment]
Thank you!

**Comment by vitawasalreadytaken** [comment]
First of all, I must apologise that we hadn't updated this open-source version of aiodebug here on GitHub for a long time. We only continued development in a private fork. I have fixed this now – the current version 2.2.1 (https://pypi.org/project/aiodebug/ & https://gitlab.com/quantlane/libs/aiodebug) is the latest version we have, and the only repository now in existence.

As part of stepping up our open-source efforts, we're gradually moving projects to GitLab.com because GitLab is what we use for internal development, and it's easier for us to just use the same tool for open-source projects as well. But I'm happy to continue the discussion for this issue here, so you don't have to open another thread over at GitLab.com.

And finally, here is the requested feature, now in code review: https://gitlab.com/quantlane/libs/aiodebug/-/merge_requests/1
